### PR TITLE
Revert #28945

### PR DESCRIPTION
**Pull request by HaoranYi**
#### Problem

Revert #28945 because it adds external dependency on 'wc'.

#### Summary of Changes

Fixes #

### CODE REVIEW

File: core/src/system_monitor_service.rs [hunk 1]
```diff
@@ -387,21 +385,25 @@ fn parse_disk_stats(reader_diskstats: &mut impl BufRead) -> Result<DiskStats, St
     Ok(stats)
 }
 
-pub struct SystemMonitorStatsReportConfig {
```
Comment (t-nelson) [inline core/src/system_monitor_service.rs:390]: mind popping the config struct out and re-introducing after merging this PR?

**Review by t-nelson — APPROVED** [review]
lgtm, thanks!

File: core/src/system_monitor_service.rs (hunk 1 above)
Comment (HaoranYi) [reply core/src/system_monitor_service.rs:390]: sure.

Comment (HaoranYi) [reply core/src/system_monitor_service.rs:390]: https://github.com/solana-labs/solana/pull/29132

**Comment by mvines** [comment]
ty for this

**Pull request merged**